src/wallet/wallet.h

```diff
@@ -505,22 +500,21 @@ class CWalletTx
             mapValueCopy["timesmart"] = strprintf("%u", nTimeSmart);
         }
 
-        std::vector<uint256> dummy_vector; //!< Used to be vMerkleBranch
-        s << tx << hashBlock << dummy_vector << nIndex;
-        std::vector<CMerkleTx> vUnused; //!< Used to be vtxPrev
-        s << vUnused << mapValueCopy << vOrderForm << fTimeReceivedIsTxTime << nTimeReceived << fFromMe << fSpent;
+        std::vector<char> dummy_vector1; //!< Used to be vMerkleBranch
+        std::vector<char> dummy_vector2; //!< Used to be vtxPrev
```

Comment:
in commit fad5dec:

I think you can keep everything in this function as is (except changing the type of the vector). This should simplify review.

Reply:
See https://github.com/bitcoin/bitcoin/pull/16451#issuecomment-515080038. I'll remove everything after the first commit if others agree.